File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Tam giác sao cân/NGOISAO4.py
n = int(input())
space = n - 1
sao = 1
for i in range(0, n):
    for j in range(0, space):
        print(" ", end="")
    for e in range(0, sao):
        print("*", end="")
    for j in range(0, space):
        print(" ", end="")
    if(sao != n * 2 - 1):
        print("")
    space -= 1
    sao += 2

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Tam giác sao cân/NGOISAO4.cs
using System;
namespace NGOISAO4
{
    public class Program
    {
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
            int space = n - 1;
            int sao = 1;
            for(int i = 0 ; i < n; i++)
            {
                for(int j = 0; j < space; j++)
                    Console.Write(" ");
                for(int e = 0; e < sao; e++)
                    Console.Write("*");
                for(int j = 0; j < space; j++)
                    Console.Write(" ");
                if(sao != n * 2 - 1)
                    Console.WriteLine("");
                space --;
                sao += 2;
            }
        }
    }
}